packages/blockchain-wallet-v4-frontend/src/data/components/simpleBuy/selectors.ts

```diff
@@ -154,6 +154,17 @@ export const getDefaultPaymentMethod = (state: RootState) => {
   return lift(transform)(sbCardsR, sbMethodsR, sbBalancesR)
 }
 
+export const hasFiatBalances = (state: RootState) => {
+  const fiatBalances = Object.keys(
+    state.components.simpleBuy.balances.data
+  ).filter(
+    currency =>
+      currency in FiatTypeEnum &&
```

Comment:
Nice! Didn't know this syntax

Reply:
Yeah, TBH I was hoping there was something in TS that was more like a set intersection so I wouldn't have to do the `filter` but I couldn't find anything. If you guys figure out a way please show me.